hack/rpm-prepare.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Reading MGC_VERSION, MCG_RELEASE and GIT_COMMIT env, if they are set"
+# Change version as needed. In most cases MGC_RELEASE would not be touched unless
+# we want to do a re-lease of same version as we are not backporting.
+export MGC_VERSION=${MGC_VERSION:=0.0.1}
+export MGC_RELEASE=${MGC_RELEASE:=1}
+
+export GIT_COMMIT=${GIT_COMMIT:=$(git rev-parse --short HEAD 2>/dev/null)}
+export MGC_RPM_VERSION=${MGC_VERSION//-}
+
+# Golang version variables, if you are bumping this, please contact redhat maintainers to ensure that internal
+# build systems can handle these versions
+export GOLANG_VERSION=${GOLANG_VERSION:-1.15}
```

Comment:
in gomod we run 1.16, I also don't see you setting that value anywhere. Any reason for 1.15 as the default?

Reply:
matter of fact, I do get a pretty weird error when I run this with 1.16:

```
[tjungblu ~/git/arjmg]$ hack/rpm-local-build.sh 
Building locally
error: Failed build dependencies:
	golang >= 1.15 is needed by must-gather-clean-0.0.1-1.el8.x86_64
[tjungblu ~/git/arjmg]$ go version
go version go1.16.4 linux/amd64

```

Reply:
Copy-pasta :)